cesium_app/handlers/prediction.py

```diff
@@ -113,10 +114,17 @@ def get(self, prediction_id=None):
             prediction_info = [p.display_info() for p in predictions]
         else:
             prediction = self._get_prediction(prediction_id)
+            prediction_info = prediction.display_info()
 
         return self.success(prediction_info)
 
     def delete(self, prediction_id):
         prediction = self._get_prediction(prediction_id)
         prediction.delete_instance()
         return self.success(action='cesium/FETCH_PREDICTIONS')
+
+
+class DownloadPredictionResultsHandler(PredictionHandler):
```

Comment:
How about we change the endpoint to `/prediction/<id>/download`?

We can also avoid creating a special handler just for this case by modifying the existing `get` for `/prediction`.